docs/use/myst.md

```diff
@@ -227,6 +227,71 @@ which automatically numbers the labelled objects.
 For example, `` {numref}`my_figure` `` will produce a reference like:
```

Comment:
Maybe these should be in their own short section like "referencing figures" where we can explain both `ref` and `numref`.